README.md

```diff
@@ -1,5 +1,10 @@
 ## HIDAPI library for Windows, Linux, FreeBSD and macOS
 
+| Build type           | Status |
```

Comment:
Doesn't seems like a "type" to me. More like a "Build group" or "CI instance" (since one CI instance may include several platforms)

Reply:
Sticked to CI instance as it describes best what's happening.